app/imdb_tools.py

```diff
@@ -17,25 +22,51 @@ def imdb_get_movies(config_path, plex, data):
         imdb_url = imdb_url[:-1]
     imdb_map = {}
     library_language = plex.Library.language
+    if "/search/" in imdb_url:
+        if "&count=" in imdb_url:
+            imdb_url = re.sub("&count=\d+", "&count=100", imdb_url)
+        else:
+            imdb_url = imdb_url + "&count=100"
     try:
         r = requests.get(imdb_url, headers={'Accept-Language': library_language})
     except requests.exceptions.MissingSchema:
         return
     tree = html.fromstring(r.content)
     title_ids = tree.xpath("//div[contains(@class, 'lister-item-image')]"
                            "//a/img//@data-tconst")
+    if "/search/" in imdb_url:
+        results = re.search('<span>\\d+-\\d+ of \\d+ titles.</span>', str(r.content))
+        total = re.findall('(\\d+)', results.group(0))[2]
+    else:
+        results = re.search('(?<=<div class="desc lister-total-num-results">).*?(?=</div>)', str(r.content))
```

Comment:
Is there a python parsing library we could leverage for this by chance? It would be great not to have to manage any HTML changes if another project is doing it already

Reply:
I'm new to python so not that I know of, but what I can say is that its already looking through the HTML of the page for the movies so if the page changes were gonna need to change something here regardless